runtimes/choose-a-renderer/overview.mdx

```diff
@@ -36,29 +36,29 @@ flutter run --no-enable-impeller
 
 ## Rive Renderer
 
-The [Rive Renderer](https://rive.app/renderer) is now available on Android and iOS. See [Specifying a Renderer](#specifying-a-renderer) to set it as your preferred renderer.
+The [Rive Renderer](https://rive.app/renderer) is now available on Android and Apple platforms. See [Specifying a Renderer](#specifying-a-renderer) to set it as your preferred renderer.
 
 While it's ready for testing and your feedback is highly valued during this phase, we advise exercising caution before considering it for production builds. You may encounter compatibility issues with certain devices. Please reach out to us on Discord or through our Support Channel.
 
 Your collaboration helps us refine and enhance the Rive Renderer to make it more robust and reliable for broader applications. Thank you for being a part of this exciting journey!
 
 <Note>
- By default, Android will use the current Skia renderer, and iOS will use the Rive Renderer (as of v6.0.0).
+ By default, Android will use the current Skia renderer, and Apple platforms will use the Rive Renderer (as of v6.0.0).
 </Note>
 
 <Tabs>
 
-    <Tab title="iOS/macOS">
+    <Tab title="Apple">
         
         ## Starting Version 
 
-        The Rive Renderer was introduced in the iOS runtime starting at **v5.3.2**, however, we recommend installing the latest version of the dependency to get the latest updates. See the [CHANGELOG](https://github.com/rive-app/rive-ios/blob/main/CHANGELOG.md) for details on the latest versions. Starting in v6.0.0, Rive Renderer is used as the default on iOS / macOS.
+        The Rive Renderer was introduced in Apple runtimes starting at **v5.3.2**, however, we recommend installing the latest version of the dependency to get the latest updates. See the [CHANGELOG](https://github.com/rive-app/rive-ios/blob/main/CHANGELOG.md) for details on the latest versions. Starting in v6.0.0, Rive Renderer is used as the default on Apple platforms.
```

Comment:
This is now already a year old. I think we can just say it was introduced in v6.0.0, but we recommend to always use the latest